jibri/Dockerfile

```diff
@@ -11,7 +11,7 @@ LABEL org.opencontainers.image.documentation="https://jitsi.github.io/handbook/"
 ARG USE_CHROMIUM=0
 #ARG CHROME_RELEASE=latest
 # https://googlechromelabs.github.io/chrome-for-testing/
-ARG CHROME_RELEASE=121.0.6167.85
+ARG CHROME_RELEASE=121.0.6167.184
```

Comment:
Can you try backing out this change?

Reply:
Yes, but this was my attempt to fix the problem. It did not solve it.

Reply:
No good I believe this is a separate problem that we need to solve

Reply:
Not sure, I built it like that 2 days ago no problem.

I see the problem seems to be the zip file layout, but it would be very odd that they changed the layout for a given version.

Reply:
Never mind, the nightly build fails the same way: https://github.com/jitsi/docker-jitsi-meet/actions/runs/7939771467

Please drop that change and I'll look into fixing it.

Reply:
It's already dropped.